ComputerGuesses.js

```diff
@@ -0,0 +1,140 @@
+//info for readline ask await functions and async functions
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+//function for generating a random integer
+function randomInteger(min, max) {
+  let range = max - min + 1;
+  return min + Math.floor(Math.random() * range);
+}
+
+//function for sanatizing responses from input
+function capitalize(word) {
+  let steralize = word.toString().trim().toLowerCase()
+  let firstLetter = steralize[0].toUpperCase()
+  let restOfWord = steralize.slice(1)
+  return firstLetter + restOfWord
+}
+
+// start async function
+start();
+
+async function start() {
+  //start off with a prompt to play the game.
+  console.log("Let's play a game where you make up a number and I try to guess it but I need you to pick the high number")
+
+  // allow the user to put the hi
+  let hiNum = await ask("So what will the highest number we can pick be? ")
+
+  // sanatize the user number to be sure its a whole number.
+  hiNum = Math.floor(parseInt(hiNum))
+
+  // if the user doesn't input a number, automatically set the max number to 100
+  if (isNaN(hiNum)) {
+    hiNum = 100
+  }
+
+  // define the low number and then use the low and the high to generate a random number for the computers first guess.
+  let lowNum = 1
+  let randNum = randomInteger(lowNum, hiNum)
+
+  //allow the user to input a number and have it wait for that to happen
+  
+  let secretNumber = await ask("Pick any number between 1 and " + hiNum + " and let me know what it is.\nI won't peek, I promise...\n");
+  secretNumber = Math.floor(parseInt(secretNumber))
+  
+  while (isNaN(secretNumber)) {
+    secretNumber = await ask("That's not a number\nCare to try again? ")
+    secretNumber = Math.floor(parseInt(secretNumber))
+  }
+  //return said secret number and say it will now guess.
+  console.log('You entered: ' + secretNumber + "\nNow let me guess!");
+
+  //Get the computer to guess a random number between 1 and the high number the user chose
+  console.log(randNum)
+
+  //give a prompt about asking if the guess was correct
+  let response = await ask("Was I right?! ")
+  
+  // this is a while loop that if the answer is not yes, it will keep generating numbers until it's right
+  while (capitalize(response).charAt(0) !== "Y") {
+  
+    // if statement that is for when the user says no (or any word starting with 'n' for that matter, try to figure out fixing this.)
```

Comment:
If you want to provide a list of different valid responses, then you could use an array and the .includes() method
```js
let noResponses = ['no', 'nah', 'nope', 'never', 'n']
if ( noResponses.includes(response.toLowerCase() ) {
    // do stuff
}
```